app/models/thread.php

```diff
@@ -65,56 +58,134 @@ public static function countAll()
         return (int) $db->value("SELECT COUNT(*) FROM thread");
     }
     
-     public function countComments()
-    {
-        $db = DB::conn();
-        return (int) $db->value("SELECT COUNT(*) FROM comment WHERE thread_id = ? ", array($this->id));
-    }
-
     public static function get($id)
     {
         $db = DB::conn();
         $row = $db->row('SELECT * FROM thread WHERE id = ?', array($id));
-
+        
         if (!$row) {
             throw new RecordNotFoundException('no record found');
         }
-
         return new self($row);
     }
 
-     public function getComments($offset, $limit)
+    public function write(Comment $comment)
```

Comment:
It seems that this method creates a comment, in that case please move this to the Comment model po :bow: